src/overlays/actors/ovl_Bg_Hidan_Dalm/z_bg_hidan_dalm.c

```diff
@@ -9,7 +9,10 @@ void BgHidanDalm_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgHidanDalm_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgHidanDalm_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void BgHidanDalm_WaitInteraction(BgHidanDalm* this, GlobalContext* globalCtx);
+void BgHidanDalm_Shrink(BgHidanDalm* this, GlobalContext* globalCtx);
+void BgHidanDalm_UpdateCollider(BgHidanDalm* this);
```

Comment:
```suggestion
```

Reply:
why?

Reply:
we only forward declare when necessary. this one is above is usage, so its not needed